.travis.yml

```diff
@@ -104,25 +113,24 @@ matrix:
   allow_failures:
     - python: "nightly"
 
-before_install:
-  - |
-    if [[ $TRAVIS_OS_NAME != 'osx' ]]; then
-      # test with non-ascii in path
-      mkdir /tmp/λ
-      export PATH=$PATH:/tmp/λ
-      export PATH=/usr/lib/ccache:$PATH
-    else
-      ci/travis/silence brew update
-      brew upgrade python
-      brew install ffmpeg imagemagick mplayer ccache
-      hash -r
-      which python
-      python --version
-      # We could install ghostscript and inkscape here to test svg and pdf
-      # but this makes the test time really long.
-      # brew install ghostscript inkscape
-      export PATH=/usr/local/opt/python/libexec/bin:/usr/local/opt/ccache/libexec:$PATH
-    fi
+before_install: |
+  # test with non-ascii in path
+  mkdir /tmp/λ
```

Comment:
I don't understand what this is doing, but why move this out of the if check?

Reply:
There used to be an error when a directory in the PATH contained non-ascii characters on Linux (something with subprocess/Py2), so to test the fix we just added such a directory to the PATH.  This patch just makes the directory also present in the OSX test, because why not?

Reply:
Maybe the comment could be expanded why we are doing this test and what we do/do not if it fails.
This is code one will only see once in a while and it‘s good to be clear about what it does.

Reply:
Actually I pushed a commit that removes this (originally introduced in #7804 for #7715).  It's a Py2-only thing which is just pointless in a Py3 world.  Milestoning accordingly.